whylog/assistant/exceptions.py

```diff
@@ -41,3 +41,24 @@ def __init__(self, assistant_class_name):
 
     def __str__(self):
         return 'This whylog version do not handle %s. Please upgrade Whylog' % self.assistant_class_name
+
+
+class SpanConstructorParamsError(WhylogError):
+    def __init__(self, start, end):
+        super(SpanConstructorParamsError, self).__init__()
```

Comment:
The `WhylogError` constructor is default class constructor. Why do you use super here?